lua/astrocommunity/note-taking/global-note-nvim/init.lua

```diff
@@ -0,0 +1,50 @@
+local get_project_name = function()
+  local project_name
+
+  local project_directory, err = vim.loop.cwd()
+  if not project_directory then
+    vim.notify(err, vim.log.levels.WARN)
+  else
+    project_name = vim.fs.basename(project_directory)
+  end
+
+  if not project_name then
+    vim.notify("Unable to get the project name", vim.log.levels.WARN)
+  end
```

Comment:
```suggestion
  if not project_name then vim.notify("Unable to get the project name", vim.log.levels.WARN) end
```

This should fix the formatting with `stylua`

Reply:
Yeah I have seen it, I have modified the commit